DU2 Docs Update (#1198)
streamr-dev/streamr-platform: app/src/docs/content/dataUnions/introToDataUnions.mdx

# Introduction to Data Unions

The Data Union framework is a data crowdsourcing/crowdselling solution built as part of the Streamr project. Working in tandem with the Streamr Network and Ethereum, the framework powers apps that enable people to earn by sharing valuable data. It enables app developers to aggregate and monetise the real-time data their app users create whilst sharing revenue with them. The Data Union is listed on the Streamr Marketplace as a type of data product for sale. 

> This process is called crowdselling. When someone subscribes to the product, the revenues, in DATA token, are automatically distributed among all of the Data Union members — the individual data producers. The framework defines various roles, which we’ll describe next.

Reviewer: ```suggestion
When someone subscribes to the product, the revenues, in DATA token, are automatically distributed among all of the Data Union members — the individual data producers. The framework defines various roles, which are described below.
```

Reviewer: crowdselling already mentioned above

As a general concept, a Data Union is a structure that allows separate entities who generate data to pool their data in order to achieve a common goal, typically to increase its value. Data Unions may differ from each other based on the membership model, use cases, revenue structure and governance, but also share some common components: a group of members contributing their data, an entity providing the application layer to manage the data and a discovery mechanism or marketplace for the data.

Streamr implements this idea with its Data Unions Framework, which allows admins to create crowdsourced data products that combine real-time data from many members into a single, more valuable dataset. 

A Data Union is an Ethereum smart contract, created and deployed via the Core app and published on the Marketplace. Most revenue from a Data Union flows to its members, but the admin can set a percentage fee they receive in return for creating and managing the union. For more information on structure, see <Link to={docsLinks.frameworkRoles}>Data Union Framework Roles</Link>.

### Smart Contracts

> Data Unions is fully on-chain and implemented as smart contracts. While the gas costs on Ethereum mainnet are currently high, Data Unions can run on any Ethereum-compatible sidechain for scalability and affordable cost of operation. The first supported sidechain is <a href="https://www.xdaichain.com" target="_blank" rel="noopener noreferrer">xDai</a>, and other options may be added in the future.

Reviewer: ```suggestion
Data Unions are fully on-chain and are implemented as smart contracts. Even if the gas costs on Ethereum mainnet are sometimes high, Data Unions can run on any Ethereum-compatible sidechain for scalability and affordable cost of operation. The first supported sidechain is <a href="https://www.xdaichain.com" target="_blank" rel="noopener noreferrer">xDai</a>, and other options may be added in the future.
```

> The benefits of having Data Unions fully on-chain are many. First of all, the security of the system becomes as strong as the security of the underlying blockchain itself. Secondly, Data Union members and builders get to benefit from composability, network effects, and services available on the same blockchain, particularly DeFi platforms.

Reviewer: ```suggestion
There are numerous benefits of having Data Unions fully on-chain. First of all, the security of the system becomes as strong as the security of the underlying blockchain itself. Secondly, Data Union members and builders get to benefit from composability, network effects, and services available on the same blockchain, particularly DeFi platforms.
```
